longhorndev/dev.go

```diff
@@ -218,7 +218,7 @@ func (d *LonghornDevice) createDev() error {
 		}
 	}
 
-	if err := util.DuplicateDevice(d.scsiDevice.Device, dev); err != nil {
+	if err := util.CreateDevice(d.scsiDevice.KernelDevice, dev); err != nil {
```

Comment:
Probably still call it `DuplicateDevice`, since considering KernelDevice.Name is in the parameter.

Reply:
The most correct name would be CreateDeviceNode?
What do you think about that?

Reply:
I meant, if you pass in a device with name into a function called `CreateDevice`, by looking at this function and it's parameters, the expectation will be: this function will create this device with this name. But that's not the case here.

Reply:
Technically we are only creating a device node, but I see what you are saying.
Will change it back to DuplicateDevice